src/main/java/timl/TimL.java

```diff
@@ -0,0 +1,104 @@
+package timl;
+
+import timl.Exeptions.EmptyException;
+import timl.Exeptions.TimException;
+import timl.task.Task;
+import timl.task.TaskManager;
+import timl.utility.Printer;
+import timl.utility.TextParser;
+
+import java.util.Scanner;
+import java.lang.String;
+
+public class TimL {
+    public static void respondToCommand(String response, Task[] list){
+        String command = TextParser.extractCommand(response);
+        String message = TextParser.extractMessage(response);
+        int emptyIndex = 0;
+        for (int i = 0; i < list.length; i++) {
+            if (list[i] == null) {
+                emptyIndex = i;
+                break;
+            }
+        }
+        int taskIndex;
+        switch (command) {
+            case "list":
+                printList(list, emptyIndex);
+                break;
+            case "mark":
+                try {
+                    taskIndex = Integer.parseInt(message) - 1;
+                    TaskManager.mark(taskIndex, list, emptyIndex);
+                }catch (NumberFormatException | TimException e){
```

Comment:
ensure the spacing for the start of the catch block is there. the layout of a try catch block is as follows:

```java
try {
    statements;
} catch (Exception exception) {
    statements;
}
```
try maintaining this spacing